examples/deployment/nginx/nginx-ingress-controller.yaml

```diff
@@ -41,11 +39,6 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
-        ports:
```

Comment:
Even if the ports aren't used (e.g. in hostNetwork), please please leave them so that the pod conflicts with itself for the right reason.

With this change, I suspect the deployment could schedule 2 on the same node and one would just crashloop fighting for the port 80 binding.